pkg/osvscanner/vulnerability_result.go

```diff
@@ -49,9 +49,10 @@ func buildVulnerabilityResults(
 
 		if rawPkg.Version != "" && rawPkg.Ecosystem != "" {
 			pkg.Package = models.PackageInfo{
-				Name:      rawPkg.Name,
-				Version:   rawPkg.Version,
-				Ecosystem: string(rawPkg.Ecosystem),
+				Name:          rawPkg.Name,
+				Version:       rawPkg.Version,
+				Ecosystem:     string(rawPkg.Ecosystem),
+				OriginLayerID: rawPkg.OriginLayerID,
```

Comment:
I noticed that the final vulnerability JSON output only includes the layer ID, but not the specific command associated with that layer. Is there any specific reason for that?
![image](https://github.com/google/osv-scanner/assets/39108850/be09be45-8a8e-45a6-b625-f680eb6fc8e5)

Reply:
Updated now to return:
- originLayerID
- Creation command
- Whether it's likely in a base image or not.